dev/com.ibm.ws.session.cache_fat_infinispan_container/fat/src/com/ibm/ws/session/cache/fat/infinispan/container/SessionCacheOneServerTest.java

```diff
@@ -422,7 +423,7 @@ public void testLastAccessedTime() throws Exception {
      * can be obtained and report statistics about the cache.
      */
     @Test
-    @SkipForRepeat({ EE9_FEATURES, EE10_FEATURES, //Needs further attention for jakartaee 9
+    @SkipForRepeat({ EE9_FEATURES, EE10_FEATURES, EE11_FEATURES, //Needs further attention for jakartaee 9
```

Comment:
There is a newer annotation that help here and avoid making changes everytime we repeat a new EE level: `SkipForRepeat.EE9_OR_LATER_FEATURES`

Reply:
changed